docs/_includes/ds6/listbox.html

```diff
@@ -0,0 +1,185 @@
+
+<div id="listbox">
+    <h2>@ebay/skin/listbox</h2>
+    <p>A listbox follows the principal of <a href="https://en.wikipedia.org/wiki/Progressive_disclosure">progressive disclosure</a>, it keeps secondary commands hidden from view until needed.</p>
+    <p>A listbox may be used as a form control. It is designed as a custom-styled component (much like a <a href="#menu">menu</a>, but different) on top of of a hidden input.</p>
+
+    <div class="demo">
+        <div class="demo__inner">
+            <span class="listbox">
+                <button class="listbox__button expand-btn" aria-haspopup="listbox1" type="button">
```

Comment:
`.listbox__button` appears to be unused in the CSS, so we can maybe leave it off for now.